playbooks/roles/ocp-config/tasks/chrony.yaml

```diff
@@ -76,3 +76,10 @@
     dest: /etc/systemd/system/chronyd.service.d/restart.conf
   notify:
     - restart chrony
+
+- name: Force Chronyd Restart 
+  systemd:
+    name: chronyd
+    daemon_reload: true
+    state: restarted
+    
```

Comment:
@schabrolles can you remove leading whitespaces on line 85 (and have it just as newline)

Reply:
You need to trim the whitespaces.